test/backend/info/technicianDetails.test.ts

```diff
@@ -0,0 +1,27 @@
+import { TechnicianDetailsBuilder, TechnicianType } from "../../../lib/backend/info/technicianDetails";
+
+describe("TechnicianDetails", () => {
+    it("should have a valid constructor", () => {
+        const TechnicianDetails = new TechnicianDetailsBuilder()
+            .withWorktype(TechnicianType.ELECTRICIAN)
+            .withResponseTime(500)
+            .withHourlyRate(500)
+            .build();
+        expect(TechnicianDetails.worktype).toBe("Electrician");
+        expect(TechnicianDetails.responseTime).toBe(500);
+        expect(TechnicianDetails.hourlyRate).toBe(500);
+
+    })
+
+    it("should have a valid constructor", () => {
+        const TechnicianDetails = TechnicianDetailsBuilder.from({
```

Comment:
same, variables should be camel case e.g. someVariableName... here is how its done: all constants that are used globally, all caps, class names Upper then camel case, function names : camel case.